src/test/java/com/wikia/webdriver/elements/mercury/pages/CategoryPage.java

```diff
@@ -39,174 +30,66 @@ public CategoryPage() {
-  public CategoryPage navigateToPageWithArticleAndWithoutMembersFromUrl() {
-    navigate.toPage(MercurySubpages.CATEGORY_WITH_ARTICLE_AND_WITHOUT_MEMBERS);
-
-    articleContainerIsNotPresent();
-    categorySectionsContainerIsNotPresent();
-    verifyErrorPage();
-
-    return this;
-  }
-
-  public CategoryPage navigateToPageWithoutArticleAndWithMembersFromUrl() {
-    navigate.toPage(MercurySubpages.CATEGORY_WITHOUT_ARTICLE_AND_WITH_MEMBERS);
-
-    articleContainerIsNotPresent();
-    categorySectionsContainerIsVisible();
-
-    return this;
-  }
-
-  public CategoryPage loadMoreMembersForSection(String name) {
-    navigateThroughSection(name, loadMoreButton);
+  public CategoryPage open(String categoryName) {
+    this.navigate.toPage(String.format("%s%s", URLsContent.WIKI_DIR, categoryName));
 
     return this;
   }
 
-  public CategoryPage loadPreviousMembersForSection(String name) {
-    navigateThroughSection(name, loadPreviousButton);
 
-    return this;
-  }
+  public ArticlePage navigateToCategoryMemberPage() {
+    WebElement member = driver.findElement(categoryMembers);
+    String memberName = member.getText();
 
-  public CategoryPage navigateToCategoryMemberPage(String name) {
-    WebElement member = driver.findElement(By.xpath(
-        String.format("//a[contains(text(), \"%s\")]", name))
-    );
+    wait.forXMilliseconds(1500);
     wait.forElementClickable(member);
     member.click();
 
     new Loading(driver).handleAsyncPageReload();
 
-    String expectedUrl = "/wiki/" + name.replaceAll(" ", "_");
-    String currentUrl = driver.getCurrentUrl();
-    Assertion.assertTrue(
-        currentUrl.contains(expectedUrl),
-        String.format("Expected part \"%s\" was not found in \"%s\".", expectedUrl, currentUrl)
-    );
-    PageObjectLogging.logInfo(String.format("You were redirected to page: \"%s\".", name));
+    PageObjectLogging.logInfo(String.format("You were redirected to page: \"%s\".", memberName));
 
-    return this;
+    return new ArticlePage();
   }
 
-  private CategoryPage articleContainerIsVisible() {
-    wait.forElementVisible(article);
-    PageObjectLogging.logInfo("Article container is visible.");
-
-    return this;
+  public String getArticleContent() {
+    return driver.findElement(articleContent).getText();
   }
 
-  private CategoryPage articleContainerIsNotPresent() {
-    wait.forElementNotPresent(article);
-    PageObjectLogging.logInfo("Article container is not present in DOM, which is nice.");
+  public boolean categoryMembersContainerIsVisible() {
+    wait.forElementVisible(categoryMembersContainer);
+    PageObjectLogging.logInfo("Category sections container is visible.");
 
-    return this;
+    return true;
   }
 
-  private CategoryPage categorySectionsContainerIsVisible() {
-    wait.forElementVisible(categorySections);
-    PageObjectLogging.logInfo("Category sections container is visible.");
+  public boolean hasCategoryMembers() {
+    wait.forElementVisible(categoryMembers);
```

Comment:
Should we log every successful assertion? We do it in the methods above but not here nor in the ones below.